telco-hub/configuration/reference-crs/required/gitops/gitopsSubscription.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: openshift-gitops-operator
   namespace: openshift-gitops-operator
 spec:
-  channel: gitops-1.13
+  channel: gitops-1.14
```

Comment:
Gitops 1.15 is the latest in the lifecycle page (1.16 is available in the catalog). Should be at least 1.15